org.eclipse.jdt.ls.tests/src/org/eclipse/jdt/ls/core/internal/handlers/InitHandlerTest.java

```diff
@@ -210,6 +213,29 @@ public void testRegisterDelayedCapability() throws Exception {
 		verify(client, times(9)).registerCapability(any());
 	}
 
+	@Test
+	public void testConfigureJVMs() throws Exception {
+		InitHandler handler = new InitHandler(projectsManager, preferenceManager, server.getClientConnection(), commandHandler);
+		IEclipsePreferences prefs = InstanceScope.INSTANCE.getNode(IConstants.PLUGIN_ID);
+		boolean isWorkspaceInitialized = isWorkspaceInitialized(prefs);
```

Comment:
variable name is same as method name, not good. Rename to `oldInitializedStatus`?

Reply:
Fixed.